src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,11 @@
+.forecast-details>* {
+  padding-block-start: 10px;
```

Comment:
I'm not sure why you've used `padding-block-start`
`padding-top` works exactly the same thing and is much more conventional